movie-mlops-project/serving/services/prediction_service.py

```
@@ -0,0 +1,340 @@
+import pandas as pd
```

Ruff (0.12.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)